etcdserver/auth/auth_test.go

```diff
@@ -73,14 +88,14 @@ func TestMergeUser(t *testing.T) {
 		},
 		{
 			User{User: "foo"},
-			User{User: "foo", Password: "$2a$10$aUPOdbOGNawaVSusg3g2wuC3AH6XxIr9/Ms4VgDvzrAVOJPYzZILa"},
```

Comment:
because we now do the hash inside merge.

Otherwise, this is not empty string, so it will recalculate the hash returning different value.